pkg/utils/utils.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/containers/image/pkg/sysregistries"
+	"github.com/containers/image/signature"
+	"github.com/sirupsen/logrus"
+)
+
+// CheckConfigFiles log messages if one or more files to set up buildah do not exist
+func CheckConfigFiles() {
```

Comment:
Should this return an error, if anyone are missing?

Reply:
yes please